src/interface/blas1_interface.hpp

```diff
@@ -764,16 +813,21 @@ typename ValueType<container_0_t>::type _sdsdot(
     const typename sb_handle_t::event_t &_dependencies) {
   constexpr bool is_usm = std::is_pointer<container_0_t>::value;
   using element_t = typename ValueType<container_0_t>::type;
-  element_t res{};
+  element_t res = element_t(0);
```

Comment:
For future reference. I don't think this change is necessary. `element_t res{}` would initialize to zero or in any case `element_t res{0}` would do it.